src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -366,13 +367,26 @@ public String serverVersion() {
 
     /**
      * Executes a HTTP request. This method provides a mechanism to extend the API
-     * <p><b>Note</b>: The response must be closed after use to release the connection.
+     * <p><b>Note</b>: Streams obtained from the HttpConnection must be closed after use to release
+     * the connection.
+     * </p>
+     * <pre>
+     * {@code
+     * HttpConnection response = account.executeRequest(Http.GET(new URL(account.getBaseUri() +
+     *         "/aNewAPI")));
+     * if (HttpURLConnection.HTTP_OK == response.getConnection().getResponseCode()) {

```

Comment:
I'm not a big fan of this `if (constant == variable)` style of doing things, it doesn't seem logical to me